worldeditadditions/utils/numbers.lua

```diff
@@ -32,11 +32,29 @@ function worldeditadditions.get_ms_time()
 end
 
 function worldeditadditions.eta(existing_times, done_count, total_count)
-    local max = 100
-    local average = worldeditadditions.average(
-        worldeditadditions.table_get_last(existing_times, max)
-    )
-    local times_left = total_count - done_count
-    if times_left == 0 then return 0 end
-    return average * times_left
+		local max = 100
+		local average = worldeditadditions.average(
+				worldeditadditions.table_get_last(existing_times, max)
+		)
+		local times_left = total_count - done_count
+		if times_left == 0 then return 0 end
+		return average * times_left
+end
+
+--- Returns the sign (+ or -) at the beginning of a string if present.
+-- @param    str        string    Input string.
+-- @param    type    string    The type of value to return. Valid values: "string" (default), "int"
+-- @return    string|int        Returns the sign string or signed multiplier (1|-1).
+function worldeditadditions.getsign(str, type)
+		if not type then type = "string" end
+		if not (type == "string" or type == "int") then
+			return false, "Error: Unknown type '"..type.."'."
+		end
+		if str:sub(1, 1) == "-" then
+				if type == "int" then return -1
+				else return "-" end
+		else
+				if type == "int" then return 1
+				else return "+" end
```

Comment:
Return values here need to return `bool, thing` rather than just `thing` because of the error handling.
```suggestion
				if type == "int" then return true, -1
				else return true, "-" end
		else
				if type == "int" then return true, 1
				else return true, "+" end
```

This may necessitate some changes elsewhere.